fleet/view/js/create_fleet.js

```diff
@@ -0,0 +1,384 @@
+var loc = window.location.pathname;
+
+var split_url = loc.split('/');
+var dir = split_url[2];
+var url_host = window.location.origin + "/" + split_url[1] + "/" + split_url[2];
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+console.log(split_host[2]);
+var index = window.location.origin + "/" + split_url[1]
+
+console.log(url_host);
+let contract_list, maintenance_list;
+
+//truck
+function change_val() {
+    var value = document.getElementById('select_truck_category').value;
+    document.getElementById('select_truck_type').innerHTML = '<option value="" selected>-Select Truck Type-</option>';
+    document.getElementById('right_driver').innerHTML = '<option value="" selected>-Select Right Driver-</option>';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
    document.getElementById("right_driver").innerHTML = '<option value="" selected>-Select Right Driver-</option>';
```